README.md

```diff
@@ -21,22 +21,24 @@ For more background, our end goal, and to let us know if you want to help, pleas
-
-1. Install a code editor, [Visual Studio Code](https://code.visualstudio.com) is recommended as it automatically installs all the extensions you need.
-2. Install [Github CLI](https://cli.github.com/) which makes it much easier to work with Git commands
-3. Create a working folder on your local computer
-4. Open a terminal and navigate to that folder using the command `cd <path/to/folder>`
-5. Type `gh repo clone mautic/user-documentation` - you may be prompted to authenticate with GitHub the first time you do this
-6. Open the folder user-documentation that is created in your editor
-7. At the bottom left of your screen you will see the default branch called 'main' is showing as your active branch. Click this, and at the top of the page will show a box where you can select 'create a new branch'. Type a name which relates to the work you plan to do. Once you do this, the bottom left will change from 'main' to the name of the branch you created. At any time you can click back into that, and change back to 'main'.
-8. Now you are working on your own branch for this change, make the changes that you need to make by editing the files, which you should see on the left pane.
-9. Use the Source Control icon on the menu on the left, to view the files you have changed. Click the plus icon next to them to 'stage' them for committing. This allows you to save files in chunks, explaining what you're doing with each chunk, rather than en-mass. It makes it easier to reverse small parts in the future.
-10. If you're making any changes to text, remember to run the commands below to update the files for translations on Transifex and include those in your PR.
-11. Once you've committed all your changes, click the Publish Branch button. This might prompt you to create a fork of the repository - that's fine.
-12. Once you've published the branch, look for the Branches section in the accordion menu under the Source Control icon. Find your branch and hover over the icon which says 'Create pull request' and click it.
-13. This will take you to the web interface of GitHub, where you can give an appropriate title and description for the changes you are proposing.
-14. If you need to make changes in the future after review, switch back to this branch as mentioned in step 7. Make the changes, and then follow steps 9-12. Update the pull request when you have completed your changes and committed and pushed them, to ask the reviewer to re-review after the updates.
+To make more complex changes, follow the steps below:
+
+1. Install a code editor. [Visual Studio Code](https://code.visualstudio.com) is recommended as it automatically installs all the extensions you need.
+2. Install [Github CLI](https://cli.github.com/) which simplifies Git commands.
+3. Create a working folder on your local computer.
+4. Open a terminal and navigate to that folder using the command `cd <path/to/folder>`.
+5. Fork the `mautic/user-documentation` repository on GitHub.
```

Comment:
```suggestion
5. Fork the `mautic/user-documentation` repository on GitHub by clicking on the fork button at the top right.
```
 I think we should give them guidance on how to actually do this because they might not know.